packages/atomic/src/components/common/item-list/styles/mixins.pcss

```diff
@@ -215,6 +215,9 @@
           grid-template-columns: repeat(3, 1fr);
         }
         @media (min-width: 1024px) {
+          grid-template-columns: repeat(3, 1fr);
+        }
+        @media (min-width: 1280px) {
           grid-template-columns: repeat(4, 1fr);
         }
```

Comment:
This new breakpoint prevents the variant icons from being too small, and other issues due to lack of space.

Reply:
Would it be better to try to fix this with a min width ?

Reply:
What do you mean? Since we are in **display: grid** a min width would cause the right column to overflow I think..

Reply:
I see! We would still end up with the other issues, 4 columns is really a lot at that width because of the facet column.